### PR TITLE
Add a build flag to disable building Unicode support

### PR DESCRIPTION
**Pull request by Catfish-Man**

### CODE REVIEW

**Comment by Catfish-Man** [comment]
@swift-ci please test

**Comment by Catfish-Man** [comment]
@swift-ci please smoke benchmark

**Comment by Catfish-Man** [comment]
@swift-ci please smoke benchmark

**Comment by Catfish-Man** [comment]
@swift-ci please test

**Review by Azoy — APPROVED** [review]
Yep, these changes look good to me!

**Comment by Catfish-Man** [comment]
@swift-ci please clean test

**Comment by Catfish-Man** [comment]
@swift-ci please clean test

**Comment by Catfish-Man** [comment]
argh I screwed up what branch I was on and that didn't have the change I meant to push

File: utils/swift_build_support/tests/products/test_swift.py [hunk 1]
```diff
@@ -62,6 +62,8 @@ def setUp(self):
             enable_experimental_concurrency=False,
             enable_experimental_distributed=False,
             build_swift_stdlib_static_print=False,
+            stdlib_unicode_data=True,
+            enable_experimental_string_processing=False,
```
Comment (Catfish-Man) [inline utils/swift_build_support/tests/products/test_swift.py:66]: This line isn't showing up in my local git diff against main and I'm so confused heh

**Comment by Catfish-Man** [comment]
@swift-ci please test and merge

**Comment by Catfish-Man** [comment]
@swift-ci please test and merge

**Comment by Catfish-Man** [comment]
@swift-ci please test and merge

**Comment by Catfish-Man** [comment]
@swift-ci please test and merge

**Comment by Catfish-Man** [comment]
@swift-ci please test and merge

**Comment by Catfish-Man** [comment]
@swift-ci please test and merge

**Comment by Catfish-Man** [comment]
@swift-ci please test and merge

**Comment by Catfish-Man** [comment]
@swift-ci please test and merge

**Pull request merged**